### PR TITLE
DOC-5372 update node shutdown with v22.2 output

### PR DESCRIPTION
**Pull request by andf-crl**
Addresses: DOC-5372

- Updates all CLI output for `cockroach node` commands to reflect running v22.2 output, now that alpha is live.
	- Obfuscated username in filepath while in here.
- Checked all other `md `files under `v22.2/` path, this is the only file that needed to be updated.

[node-shutdown.md](https://deploy-preview-15000--cockroachdb-docs.netlify.app/docs/v22.2/node-shutdown)

### CODE REVIEW

File: v22.2/node-shutdown.md [hunk 1]
```diff
@@ -523,16 +523,16 @@ To drain and shut down a node that was started in the foreground with [`cockroac
     ~~~
 
     ~~~
-    CockroachDB node starting at 2022-02-11 06:25:24.922474 +0000 UTC (took 5.1s)
-    build:               CCL v22.1.0-791-ga0f0df7927 @ 2022/02/02 20:08:24 (go1.17.6)
+    CockroachDB node starting at 2022-09-01 06:25:24.922474 +0000 UTC (took 5.1s)
```
Comment (nickvigilante) [inline v22.2/node-shutdown.md:526]:
```suggestion
    CockroachDB node starting at {{ now | date: "%Y-%m-%d %H:%M:%S.%6 +0000 UTC" }} (took 5.1s)
```

File: v22.2/node-shutdown.md [hunk 2]
```diff
@@ -523,16 +523,16 @@ To drain and shut down a node that was started in the foreground with [`cockroac
...
+    build:               CCL v22.2.0-alpha.1 @ 2022/08/30 23:02:58 (go1.18.4)
```
Comment (nickvigilante) [inline v22.2/node-shutdown.md:527]:
```suggestion
    build:               CCL {{page.release_info.version}} @ {{page.release_info.build_time}}
```

**Review by nickvigilante — APPROVED** [review]
Approved with 2 changes.

**Comment by andf-crl** [comment]
Ah, looks like we can't quite be this clever. Build time and go version don't presently match between actual command output and referenced CSV values. For now, let's merge this hardcoded to v22.2, and revisit possible automation enhancements later. Love the idea though!!

**Pull request merged**